pkg/identityserver/user_registry.go

```diff
@@ -77,11 +78,11 @@ func (is *IdentityServer) validatePasswordStrength(ctx context.Context, password
 	var uppercase, digits, special int
 	for _, r := range password {
 		switch {
-		case r >= 'A' && r <= 'Z':
+		case unicode.IsUpper(r):
 			uppercase++
-		case r >= '0' && r <= '9':
+		case unicode.IsDigit(r):
 			digits++
-		case r == '!' || r == '%' || r == '@' || r == '#' || r == '$' || r == '&' || r == '*':
+		case !unicode.IsLetter(r) && !unicode.IsNumber(r):
```

Comment:
This includes category `Z`, separators(e.g. ` `). IMO it makes sense to count space etc. as special, by looking at https://github.com/TheThingsNetwork/lorawan-stack/issues/1223#issuecomment-524226731 seems that @htdvisser as well.

Reply:
Another possiblity is to be more strict and count `S`, `P`, `M`, `Z` here, for example, but IMO the current solution is best.

Reply:
We can also just catch all "other" `IsPrint` here.